ocp_resources/pipelineruns.py

```diff
@@ -0,0 +1,74 @@
+# API reference: https://tekton.dev/docs/pipelines/pipelineruns/
+
+from ocp_resources.resource import NamespacedResource
+
+
+class PipelineRun(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.TEKTON_DEV
+
+    def __init__(
+        self,
+        name,
+        namespace,
+        win_image_download_url,
```

Comment:
`win_image_download_url` is a specific use case of what params can get; params should get a list of dicts with  `name` and `value` keys